features/steps/sample_steps.py

```diff
@@ -29,8 +30,12 @@ def wait_for_full_sample_ingest(context):
     _wait_for_queue_to_be_drained(Config.RABBITMQ_SAMPLE_INBOUND_QUEUE)
     _wait_for_queue_to_be_drained(Config.RABBITMQ_SAMPLE_TO_ACTION_QUEUE)
     context.sample_fully_ingested_time = datetime.utcnow()
-    print(f'Time to fully ingest sample into action scheduler: '
-          f'[{str(context.sample_fully_ingested_time - context.sample_load_start_time)}]\n')
+    time_taken = context.sample_fully_ingested_time - context.sample_load_start_time
+    time_taken_metric = json.dumps({
+        'event': 'Time to fully ingest sample into action scheduler',
```

Comment:
Can I be a real fusspot and suggest that this might be better as `event_description` and then we use short identifiers for each event 'type' like `SAMPLE_INGEST_TO_ACTION_CASES` to make it easier to graph certain things, and we can change the descriptions without breaking anything.